### PR TITLE
check on rasters + fix RCM bug

### PR DESCRIPTION
**Pull request by vincelhx**
self.rasters needs to be an instance attr ; not class attr

image

### CODE REVIEW

**Comment by vincelhx** [comment]
i also renamed  `model_vv` => `model_co` et `model_vh` =>  `model_cross`

File: src/xsar/rcm_meta.py [hunk 1]
```diff
@@ -103,7 +106,7 @@ def flip_sample_da(self):
         antenna_pointing = self.dt['sourceAttributes/radarParameters'].attrs['antennaPointing']
         pass_direction = self.dt['sourceAttributes/orbitAndAttitude/orbitInformation'].attrs['passDirection']
         flipped_cases = [('Left', 'Ascending'), ('Right', 'Descending')]
-        samples_depending_ds = ['geolocationGrid', 'lut', 'noise_lut']
+        samples_depending_ds = ['geolocationGrid']
```
Comment (agrouaze) [inline src/xsar/rcm_meta.py:109]: Is it the modification tha prevent the bug of flipped RCM images? Is there an issue opened related to this problem?

Comment (vincelhx) [reply src/xsar/rcm_meta.py:109]: No issue, should I create one to explain the problem? 
Yes, this is the line that prevent the RCM bug

Comment (agrouaze) [reply src/xsar/rcm_meta.py:109]: note: the variable name `samples_depending_ds` is not super explicit.

**Comment by agrouaze** [comment]
@vincelhx I doing an infringement to our rules of githuber, I set your PR as "ready for review" and I merged it but it seems that Théo is stuck with his processings without your correction on RCM. 
Unit test is still needed and the second issue regarding the bias on the incidence will be done separately. Thanks.

**Pull request merged**